crates/autopilot/src/driver_model.rs

```diff
@@ -85,6 +85,8 @@ pub mod execute {
     #[serde(rename_all = "camelCase")]
     pub struct Request {
         pub auction_id: i64,
+        #[serde(with = "bytes_hex")]
+        pub tag: Vec<u8>,
```

Comment:
Does this get `Debug` printed as something readable?

Reply:
It gets debug printed like all integer vectors: `vec[0, 1, 2, 3]`. I think it's fine because we don't debug this struct and the tag is short. Doesn't feel worth the effort of a manual Debug implementation.

Reply:
This could be derived with:
```rust
    #[derive(Clone, Debug, Default, Deserialize, Serialize)]
    #[serde(rename_all = "camelCase")]
    #[derivative(Debug)]
    pub struct Request {
        pub auction_id: i64,
        #[derivative(Debug(format_with = "shared::debug_bytes"))]
        #[serde(with = "bytes_hex")]
        pub tag: Vec<u8>,
    }
```